### PR TITLE
Import new splash screen and systray-icon

### PR DESCRIPTION
**Pull request by zander**
Artwork by Mick Bruce mickbruce.mjb@gmail.com

### CODE REVIEW

**Comment by dskloet** [comment]
Why does (so much) code need to be changed just to change an image?

**Comment by christophebiocca** [comment]
Tested and it works (ArchLinux; XFCE4).

~~@dskloet: My guess is that he modified the 0.12 version of the file, and is now backporting those code changes inadvertently?~~

**Comment by zander** [comment]
I didn't touch the 0.12 branch.

The reason for the amount of code is based on the differences in content and some bad code in the old version.

We now have a individual images for splash and (systray) icon. Both needed to be colourized individually, hence some refactor.
At the same time the usage of wrong classes (QICon and QPixmap) was removed to avoid lots of copying and converting of data. The main effect of that was that I could now use a proper painting method so resizing the splash to the target display DPI looks good.

**Comment by rromanchuk** [comment]
Tested ACK (with 0.12 branch)

**Comment by rromanchuk** [comment]
@zander just a pet peeve of mine, while you're in this area, anyway you can change the tooltip from bitcoin-qt (this is confusing engineer speak to the end user) to Bitcoin Classic, the name of the application they are actually running.

**Comment by jtoomim** [comment]
utACK.

**Comment by jgarzik** [comment]
Looks good from a quick scan.  Will test this weekend, hopefully merge this weekend.

Nice work!

**Comment by zander** [comment]
Rebased and added new copyright year.

People asked what it looks like;
splash
about

The old code still adjusts for the end-users aspect ratio based on DPI (mine is 96x96).

**Comment by pdpi** [comment]
LGTM.

**Comment by zander** [comment]
Rebased to fix merge conflict in configure.ac

**Comment by sandakersmann** [comment]
utACK

**Comment by zander** [comment]
I'll revisit for a future branch.

File: src/clientversion.h [hunk 1]
```diff
@@ -38,7 +38,7 @@
 #define DO_STRINGIZE(X) #X
 
 //! Copyright string used in Windows .rc files
-#define COPYRIGHT_STR "2009-" STRINGIZE(COPYRIGHT_YEAR) " The Bitcoin Core Developers"
+#define COPYRIGHT_STR "2009-" STRINGIZE(COPYRIGHT_YEAR) " The Bitcoin Developers"
```
Comment (hikari-no-yume) [inline src/clientversion.h:41]: This kinda strips both Core and Classic of attribution. Perhaps you should attribute both of them? Leave the Core message intact, and add another for Classic.

**Pull request closed without merging**